dev/parm/config/gfs/config.resources.AWSPW

```diff
@@ -3,40 +3,97 @@
 
 # AWS-specific job resources
 
-export is_exclusive="True"
 unset memory
 unset "memory_${RUN}"
-export threads_per_task=1
 
 step=$1
 
 case ${step} in
+  "prep")
+    export PARTITION_BATCH="process"
+    export ntasks=12
+    export tasks_per_node=12
+    ;;
+
   "fcst" | "efcs" | "wavepostpnt")
-    export PARTITION_BATCH="compute"
-    unset PARTITION_SERVICE
-    max_tasks_per_node=48
-    tasks_per_node=48
+    export tasks_per_nodes=48
```

Comment:
Suggest changing this to `npe_node_max` so that you can more easily change your target compute node type.  `npe_node_max` is defined in `config.resources`, btw.

```suggestion
    export tasks_per_nodes=${npe_node_max}
```